SI/Sem.02/Pract.02/Task03.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+
+int main() {
+	double number1 = 0, number2 = 0;
+
+	std::cout << "Enter first number value: ";
+	std::cin >> number1;
+	std::cout << "Enter second number value: ";
+	std::cin >> number2;
+
+	std::cout << (number1 > number2 ? "The first number is larger than the second number." : "The second number is larger than the first number.") << std::endl;
```

Comment:
if number1 == number2 ?

Reply:
Well the in the exercise it is said that the "program accepts 2 different numbers". I understand that as "it's the user's responsibility to input 2 different numbers, BECAUSE the program is said to take in 2 different numbers. That's how interpret it, just by reading. Of course, I'm saying I'm right, just that the text is a bit misleading. I will fix it though. :D

Reply:
True that.
Either way, we should check for sure that they are different.